spec/app/domain/authentication/authn-oidc/v2/views/providor_context_spec.rb

```diff
@@ -19,52 +18,66 @@
   let(:endpoint) { double(authorization_endpoint: '"http://test"') }
 
   let(:foo) do
-    Authentication::AuthnOidc::V2::DataObjects::Authenticator
-      .new(account: "cucumber",
-           service_id: "foo",
-           redirect_uri: "http://conjur/authn-oidc/cucumber/authenticate",
-           provider_uri: "http://test",
-           name: "foo",
-           state: 'foostate',
-           client_id: "ConjurClient",
-           client_secret: 'client_secret',
-           claim_mapping: 'claim_mapping',
-           nonce: 'secret',
-           )
+    Authentication::AuthnOidc::V2::DataObjects::Authenticator.new(
+      account: "cucumber",
+      service_id: "foo",
+      redirect_uri: "http://conjur/authn-oidc/cucumber/authenticate",
+      provider_uri: "http://test",
+      name: "foo",
```

Comment:
Would it be better to have name and service_id be different to ensure that the values never get crossed? Possibly an unlikely bug, but if it happened, this test would miss it. Real-life debugging might be enough though to catch it.

Reply:
I've [made `service_id` and `name` distinct](https://github.com/cyberark/conjur/pull/2662/commits/b0955cc41a4f1577414acac95ba8cfa1f6761d91#diff-beac8fad21a2028169ab07cd7601c44fe2b5caba747f4f48921b897f6202e785R23) in the latest commit.